src/jsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "typeAcquisition": {
+      "include": [
+          "jest"
+      ]
+  }
+}
```

Comment:
[This is needed to have autocompletion with vsCode](https://humanwhocodes.com/snippets/2019/05/jest-globals-intellisense-visual-studio-code/)